src/wx/vbam-options-static.cpp

```diff
@@ -0,0 +1,830 @@
+#include "vbam-options.h"
+
+// Helper implementation file to define and compile all of these huge constants
+// separately. These should not be updated very often, so this improves 
+
+#include <algorithm>
+#include <wx/log.h>
+
+#include "../common/ConfigManager.h"
+#include "../gb/gbGlobals.h"
+#include "opts.h"
+#include "wx/stringimpl.h"
+
+#define VBAM_OPTIONS_INTERNAL_INCLUDE
+#include "vbam-options-internal.h"
+#undef VBAM_OPTIONS_INTERNAL_INCLUDE
+
+namespace {
+
+// This enum must be kept in sync with the one in wxvbam.h
```

Comment:
We'll need to unify these, either now or later, duplication is bad.

Perhaps the string list could be taken from the xrc and the enum could be generated from the xrc.